src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -72,6 +68,12 @@ var autoStartRbacRoles = [
   '673868aa-7521-48a0-acc6-0f60742d39f5' // Data Factory contributor - https://learn.microsoft.com/azure/role-based-access-control/built-in-roles#data-factory-contributor
 ]
 
+var storageRbacRoles = [
```

Comment:
Do you want to merge the scripts into one pre-deployment script? Not sure if everything can run before... 🤔 (Probably better in a separate PR.)